run_stargz_test.go

```diff
@@ -0,0 +1,41 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"testing"
+
+	"github.com/containerd/nerdctl/pkg/testutil"
+)
+
+func TestRunStargz(t *testing.T) {
+	testutil.DockerIncompatible(t)
+	base := testutil.NewBase(t)
+	requiresStargz(base)
+	// if stargz snapshotter is functional, "/.stargz-snapshotter" appears
+	base.Cmd("--snapshotter=stargz", "run", "--rm", testutil.FedoraESGZImage, "ls", "/.stargz-snapshotter").AssertOK()
+}
```

Comment:
Thanks for this. Can we also add a check if all snapshots are prepared as remote snapshots by Stargz Snapshotter? Stargz Snapshotter adds a label `containerd.io/snapshot/remote` for each remote snapshot so we can check it using this label.

e.g.

https://github.com/containerd/stargz-snapshotter/blob/17b648dd9cd4a4b859762c07b85d302e166c3387/script/pullsecrets/create-pod.sh#L84-L105

Reply:
We don't have CLI (`nerdctl image inspect --mode=native`) for that yet

Reply:
Can we merge this PR as-is?

Reply:
Yes, we can.